modules/mobile/app/controllers/mobile/v0/payment_information_controller.rb

```diff
@@ -1,40 +1,55 @@
 # frozen_string_literal: true
 
 require 'evss/ppiu/service'
+require 'lighthouse/direct_deposit/client'
+require 'lighthouse/direct_deposit/error_parser'
+require 'lighthouse/direct_deposit/payment_account'
 require_relative '../concerns/sso_logging'
 
 module Mobile
   module V0
     class PaymentInformationController < ApplicationController
       include Mobile::Concerns::SSOLogging
 
-      before_action { authorize :evss, :access? }
-      before_action { authorize :ppiu, :access? }
+      before_action { authorize :evss, :access? unless lighthouse? }
+      before_action { authorize :ppiu, :access? unless lighthouse? }
+      before_action { authorize :lighthouse, :access_update? if lighthouse? }
```

Comment:
So it should be access_update for the index as well? I had assumed access_update was for posting changes, not for getting data.

Reply:
You are correct, and the code is confusing. There is already an existing `access?` function that does NOT match the same `access?` rules of the evss equivalent. The `access_update?` that I added DOES match those rules. I can create another function that explains this further, but it would be the exact same function duplicated. I can also just add an alias so that it reads better

Reply:
I just aliased the method so that it hopefully makes more sense